ttnn/ttnn/operations/relational.py

```diff
@@ -159,6 +159,7 @@ def relational_function(
                     output_mem_config=memory_config,
                 )
             )
+            input_tensor_b = ttnn.to_layout(input_tensor_b, ttnn.TILE_LAYOUT)
```

Comment:
Please don't add this in `ttnn` source code and update the tests accordingly instead

Reply:
done. updated tensor creation with layout and device.